solidus_social.gemspec

```diff
@@ -39,6 +39,8 @@ Gem::Specification.new do |spec|
   spec.add_dependency 'solidus_auth_devise'
   spec.add_dependency 'solidus_core', ['>= 2.0.0', '< 4']
   spec.add_dependency 'solidus_support', '~> 0.5'
+  spec.add_dependency 'multi_json'
```

Comment:
I can't see any reference to a `MultiJson` constant. Have we tried just removing the dependency?

Reply:
The install fails with the following error when `multi_json` is removed
```
cannot load such file -- multi_json (LoadError)
```

Reply:
Could you provide the entire stack of the error? :pray:

Reply:
```
Traceback (most recent call last):
	42: from bin/rails:4:in `<main>'
	41: from /Users/piyushswain/.asdf/installs/ruby/2.7.4/lib/ruby/gems/2.7.0/gems/bootsnap-1.12.0/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:30:in `require'
	40: from /Users/piyushswain/.asdf/installs/ruby/2.7.4/lib/ruby/gems/2.7.0/gems/bootsnap-1.12.0/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:30:in `require'
	39: from /Users/piyushswain/.asdf/installs/ruby/2.7.4/lib/ruby/gems/2.7.0/gems/railties-7.0.3.1/lib/rails/commands.rb:18:in `<main>'
	38: from /Users/piyushswain/.asdf/installs/ruby/2.7.4/lib/ruby/gems/2.7.0/gems/railties-7.0.3.1/lib/rails/command.rb:48:in `invoke'
	37: from /Users/piyushswain/.asdf/installs/ruby/2.7.4/lib/ruby/gems/2.7.0/gems/railties-7.0.3.1/lib/rails/command/base.rb:87:in `perform'
	36: from /Users/piyushswain/.asdf/installs/ruby/2.7.4/lib/ruby/gems/2.7.0/gems/thor-1.2.1/lib/thor.rb:392:in `dispatch'
	35: from /Users/piyushswain/.asdf/installs/ruby/2.7.4/lib/ruby/gems/2.7.0/gems/thor-1.2.1/lib/thor/invocation.rb:127:in `invoke_command'
	34: from /Users/piyushswain/.asdf/installs/ruby/2.7.4/lib/ruby/gems/2.7.0/gems/thor-1.2.1/lib/thor/command.rb:27:in `run'
	33: from /Users/piyushswain/.asdf/installs/ruby/2.7.4/lib/ruby/gems/2.7.0/gems/railties-7.0.3.1/lib/rails/commands/console/console_command.rb:101:in `perform'
	32: from /Users/piyushswain/.asdf/installs/ruby/2.7.4/lib/ruby/gems/2.7.0/gems/railties-7.0.3.1/lib/rails/command/actions.rb:14:in `require_application_and_environment!'
	31: from /Users/piyushswain/.asdf/installs/ruby/2.7.4/lib/ruby/gems/2.7.0/gems/railties-7.0.3.1/lib/rails/command/actions.rb:22:in `require_application!'
	30: from /Users/piyushswain/.asdf/installs/ruby/2.7.4/lib/ruby/gems/2.7.0/gems/bootsnap-1.12.0/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:30:in `require'
	29: from /Users/piyushswain/.asdf/installs/ruby/2.7.4/lib/ruby/gems/2.7.0/gems/bootsnap-1.12.0/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:30:in `require'
	28: from /Users/piyushswain/nebulab/solidus_social_test_app/config/application.rb:7:in `<main>'
	27: from /Users/piyushswain/.asdf/installs/ruby/2.7.4/lib/ruby/gems/2.7.0/gems/bundler-2.3.6/lib/bundler.rb:176:in `require'
	26: from /Users/piyushswain/.asdf/installs/ruby/2.7.4/lib/ruby/gems/2.7.0/gems/bundler-2.3.6/lib/bundler/runtime.rb:44:in `require'
	25: from /Users/piyushswain/.asdf/installs/ruby/2.7.4/lib/ruby/gems/2.7.0/gems/bundler-2.3.6/lib/bundler/runtime.rb:44:in `each'
	24: from /Users/piyushswain/.asdf/installs/ruby/2.7.4/lib/ruby/gems/2.7.0/gems/bundler-2.3.6/lib/bundler/runtime.rb:55:in `block in require'
	23: from /Users/piyushswain/.asdf/installs/ruby/2.7.4/lib/ruby/gems/2.7.0/gems/bundler-2.3.6/lib/bundler/runtime.rb:55:in `each'
	22: from /Users/piyushswain/.asdf/installs/ruby/2.7.4/lib/ruby/gems/2.7.0/gems/bundler-2.3.6/lib/bundler/runtime.rb:60:in `block (2 levels) in require'
	21: from /Users/piyushswain/.asdf/installs/ruby/2.7.4/lib/ruby/gems/2.7.0/gems/bootsnap-1.12.0/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:30:in `require'
	20: from /Users/piyushswain/.asdf/installs/ruby/2.7.4/lib/ruby/gems/2.7.0/gems/bootsnap-1.12.0/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:30:in `require'
	19: from /Users/piyushswain/nebulab/solidus_social/lib/solidus_social.rb:8:in `<main>'
	18: from /Users/piyushswain/.asdf/installs/ruby/2.7.4/lib/ruby/gems/2.7.0/gems/polyglot-0.3.5/lib/polyglot.rb:65:in `require'
	17: from /Users/piyushswain/.asdf/installs/ruby/2.7.4/lib/ruby/gems/2.7.0/gems/bootsnap-1.12.0/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:30:in `require'
	16: from /Users/piyushswain/.asdf/installs/ruby/2.7.4/lib/ruby/gems/2.7.0/gems/bootsnap-1.12.0/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:30:in `require'
	15: from /Users/piyushswain/nebulab/solidus_social/lib/solidus_social/engine.rb:7:in `<main>'
	14: from /Users/piyushswain/.asdf/installs/ruby/2.7.4/lib/ruby/gems/2.7.0/gems/polyglot-0.3.5/lib/polyglot.rb:65:in `require'
	13: from /Users/piyushswain/.asdf/installs/ruby/2.7.4/lib/ruby/gems/2.7.0/gems/bootsnap-1.12.0/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:30:in `require'
	12: from /Users/piyushswain/.asdf/installs/ruby/2.7.4/lib/ruby/gems/2.7.0/gems/bootsnap-1.12.0/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:30:in `require'
	11: from /Users/piyushswain/.asdf/installs/ruby/2.7.4/lib/ruby/gems/2.7.0/gems/omniauth-amazon-1.0.1/lib/omniauth-amazon.rb:1:in `<main>'
	10: from /Users/piyushswain/.asdf/installs/ruby/2.7.4/lib/ruby/gems/2.7.0/gems/polyglot-0.3.5/lib/polyglot.rb:65:in `require'
	 9: from /Users/piyushswain/.asdf/installs/ruby/2.7.4/lib/ruby/gems/2.7.0/gems/bootsnap-1.12.0/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:30:in `require'
	 8: from /Users/piyushswain/.asdf/installs/ruby/2.7.4/lib/ruby/gems/2.7.0/gems/bootsnap-1.12.0/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:30:in `require'
	 7: from /Users/piyushswain/.asdf/installs/ruby/2.7.4/lib/ruby/gems/2.7.0/gems/omniauth-amazon-1.0.1/lib/omniauth/amazon.rb:2:in `<main>'
	 6: from /Users/piyushswain/.asdf/installs/ruby/2.7.4/lib/ruby/gems/2.7.0/gems/polyglot-0.3.5/lib/polyglot.rb:65:in `require'
	 5: from /Users/piyushswain/.asdf/installs/ruby/2.7.4/lib/ruby/gems/2.7.0/gems/bootsnap-1.12.0/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:30:in `require'
	 4: from /Users/piyushswain/.asdf/installs/ruby/2.7.4/lib/ruby/gems/2.7.0/gems/bootsnap-1.12.0/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:30:in `require'
	 3: from /Users/piyushswain/.asdf/installs/ruby/2.7.4/lib/ruby/gems/2.7.0/gems/omniauth-amazon-1.0.1/lib/omniauth/strategies/amazon.rb:2:in `<main>'
	 2: from /Users/piyushswain/.asdf/installs/ruby/2.7.4/lib/ruby/gems/2.7.0/gems/polyglot-0.3.5/lib/polyglot.rb:65:in `require'
	 1: from /Users/piyushswain/.asdf/installs/ruby/2.7.4/lib/ruby/gems/2.7.0/gems/bootsnap-1.12.0/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:15:in `require'
/Users/piyushswain/.asdf/installs/ruby/2.7.4/lib/ruby/gems/2.7.0/gems/bootsnap-1.12.0/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:15:in `require': cannot load such file -- multi_json (LoadError)
```

Reply:
I see. It's a problem with amazon's Omniauth integration. I created a [PR](https://github.com/wingrunr21/omniauth-amazon/pull/19), but it doesn't look to be very well maintained.